api/dp-openapi.yaml

```diff
@@ -1128,6 +1157,26 @@ components:
           type: string
           format: dateTime
 
+    project_transition:
+      required:
+      - timestamp
+      - event_type
+      - details
+      type: object
+      properties:
+        timestamp:
+          type: string
+          format: dateTime
+        event_type:
+          type: string
+          enum: [creation, transition]
+        details:
+          type: array
```

Comment:
I think these objects take a `description` and it might be worth adding something like:
```
description: Additional details about the transition.
  The schema of what is contained in these strings is undefined
  and the contents may change.
```

Reply:
Done.